solutions/src/main/java/ru/sberbank/school/task08/SerializableManager.java

```diff
@@ -19,41 +18,30 @@ public SerializableManager(@NonNull String filesDirectoryPath) {
 
     @Override
     public void initialize() {
-//        numberOfFile +=1 ;
-//        InstantiatableEntity game = createEntity(InstantiatableEntity.Type.BUILDING,
-//                InstantiatableEntity.Status.DESPAWNED, numberOfFile);
-//        List<InstantiatableEntity> listEntitites = new ArrayList<>();
-//        Savable<InstantiatableEntity> mapSavedEntitites = createSavable("saveGame"+numberOfFile, listEntitites);
-//        System.out.println("Before serialization: " + game);
-//        String filename = filesDirectory.concat("saveGame"+numberOfFile+".bin");
-//        try {
-//            saveGame(filename, mapSavedEntitites);
-//            Savable loadedData = loadGame(filename);
-//            System.out.println("After deserialization: " + loadedData);
-//        } catch (SaveGameException ex) {
-//
-//        }
-
     }
 
     @Override
-    public void saveGame(String filename, MapState<GameObject> gameState) throws SaveGameException {
+    public void saveGame(@NonNull String filename, @NonNull MapState<GameObject> gameState) throws SaveGameException {
         try (FileOutputStream fos = new FileOutputStream(filesDirectory + File.separator + filename);
              ObjectOutputStream out = new ObjectOutputStream(fos)) {
```

Comment:
&#39;try resource&#39; has incorrect indentation level 13, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a57091964>*